tools/add_triage_signature.py

```diff
@@ -1943,6 +1943,30 @@ def _get_interfaces(self, host: dict):
 
         return interfaces_details
 
+class NetworksMtuMismatch(ErrorSignature):
+    LOG_PATTERN = re.compile(r'Failed to start sdn: interface MTU [(]([0-9]+)[)]'
+                             r' is too small for specified overlay MTU [(]([0-9]+)[)]')
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Networks MTU Mismatch",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        try:
+            sdn_logs = get_triage_logs_tar(
+                triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"]
+            ).get("controller_logs.tar.gz/must-gather.tar.gz/must-gather.local.*/*/namespaces/openshift-sdn/pods/sdn-*/sdn/sdn/logs/*.log")
+        except FileNotFoundError:
+            return
+        match = self.LOG_PATTERN.search(sdn_logs)
+        if match:
+            self._update_triaging_ticket("SDN failed to start: Overlay (cluster) network MTU {}"
+                                         " is bigger than the interface MTU {}".format(match[2], match[1]))
```

Comment:
nit: can be simplified

```suggestion
        if match := self.LOG_PATTERN.search(sdn_logs):
            self._update_triaging_ticket(f"SDN failed to start: Overlay (cluster) network MTU {match[2]}"
                                         " is bigger than the interface MTU {match[1]}")
```

Reply:
OK